### ISSUE TITLE
Problem editing information on review screen User: bug

### ISSUE THREAD

**Issue by suffolklitlab-issues** [SuffolkLITLab/docassemble-AppealsCivilDocketingStatement#21]
&nbsp; | &nbsp;
-------|------------------------------------
Question ID | `civil_docketing_statement review screen`
Steps to reproduce | Tried to go back and edit an answer (party name)
Expected behavior | let me change answer
What happened instead | Clicked the green EDIT button next to field I wanted to edit and nothing happened. (Using Google Chrome browser)
Other information | it would be helpful to offer the full and complete form in blank so user either can download and print that to complete manually or with other applications, OR, so user can see where the form Q&A is heading and know what is being requested so they give best answers
Variable being sought | `review_civil_docketing_statement`
Package version | `0.0.5`

**Comment by nonprofittechy** [comment]
@BryceStevenWilley is this still your form? It would be helpful to know which field the user had trouble editing, but this is worth taking a look at

**Comment by BryceStevenWilley** [comment]
Yep, I think I've fixed it in https://github.com/SuffolkLITLab/docassemble-AppealsCivilDocketingStatement/tree/review_fixes, I just need to throw it on the Test server and make sure it holds up.

Some of the changes could be moved up to AssemblyLine, specifically the default revisit screen for ALPeopleList objects, and maybe the default tables for ALIndividuals?

Also, not quite sure how we should address the second comment:
> it would be helpful to offer the full and complete form in blank so user either can download and print that to complete manually or with other applications, OR, so user can see where the form Q&A is heading and know what is being requested so they give best answers

I made [an issue on AssemblyLine ](https://github.com/SuffolkLITLab/docassemble-AssemblyLine/issues/260#issue-958383431) with my thoughts, it fits better there.
